GeneratorInterface/Pythia8Interface/plugins/PowhegHooksBB4L.h

```diff
@@ -1,329 +1,511 @@
 // PowhegHooksBB4L.h 
-
-// Author: Tomas Jezo, Markus Seidel and Ben Nachman based on 
-// PowhegHooks.h by Richard Corke
+// Copyright (C) 2017 Silvia Ferrario Ravasio, Tomas Jezo, Paolo Nason, Markus Seidel
```

Comment:
@tommasoboccali @sextonkennedy @davidlange6 I do not want to play the lawyer, but is there any implication by such a statement? "Authors" is one thing, I am not sure what does a Copyright means in this context

@intrepid42 you are in the list...

Reply:
In principle this should just follow the usual Powheg license, or not?
* http://powhegbox.mib.infn.it/AAAREADME-LICENSE
* http://powhegbox.mib.infn.it/MCNET-GUIDELINES

Better check with Tomas if we can just change it to this:
```
// Provided by Powheg bb4l authors: Silvia Ferrario Ravasio, Tomas Jezo, Paolo Nason
```

Reply:
@intrepid42 thanks  read in the license text "The copyright license of the POWHEG software is the GNU General Public
License (GPL) version 2." so I assume it should be ok, I let the others comment in case.